modules/juce_core/native/juce_win32_SystemStats.cpp

```diff
@@ -60,7 +60,8 @@ static void callCPUID (int result[4], uint32 type)
 #else
 static void callCPUID (int result[4], int infoType)
 {
-   #if JUCE_PROJUCER_LIVE_BUILD
+   #if JUCE_PROJUCER_LIVE_BUILD || defined(_M_ARM) || defined(_M_ARM64)
+    (void)infoType;

```

Comment:
The usual JUCE way is to make use of [`ignoreUnused`](https://github.com/juce-framework/JUCE/blob/develop/modules/juce_core/maths/juce_MathsFunctions.h#L286-L289) in these situations: `ignoreUnused (infoType);`